src/L2_projection.jl

```diff
@@ -71,21 +67,22 @@ function _mass_qr(::Lagrange{shape, 2}) where {shape <: RefSimplex}
 end
 _mass_qr(ip::VectorizedInterpolation) = _mass_qr(ip.ip)
 
-function Base.show(io::IO, ::MIME"text/plain", proj::L2Projector)
-    println(io, typeof(proj))
-    println(io, "  projection on:           ", length(proj.set), "/", getncells(get_grid(proj.dh)), " cells in grid")
-    println(io, "  function interpolation:  ", proj.func_ip)
-    println(io, "  geometric interpolation: ", proj.geom_ip)
-end
-
-function _assemble_L2_matrix(fe_values, set, dh)
-
-    n = Ferrite.getnbasefunctions(fe_values)
+function _assemble_L2_matrix(dh::DofHandler, qrs_lhs::Vector{<:QuadratureRule})
     M = create_symmetric_sparsity_pattern(dh)
     assembler = start_assemble(M)
+    for (sdh, qr_lhs) in zip(dh.subdofhandlers, qrs_lhs)
+        ip_fun = only(sdh.field_interpolations)
+        ip_geo = default_interpolation(getcelltype(sdh))
+        cv = CellValues(qr_lhs, ip_fun, ip_geo)
```

Comment:
Would it possible to pass `update_detJdV=true` and `update_gradient = false` here ?